solana/devnet_setup.sh

```diff
@@ -30,10 +30,14 @@ cli_address=6sbzC1eH4FTujJXWj51eQe25cYvr4xfXbJ1vAj7j2k5J
 bridge_address=Bridge1p5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o
 nft_bridge_address=NFTWqJR8YnRVqPDvTJrYuLrQDitTG5AScqbeghi4zSA
 token_bridge_address=B6RHG3mfcckmrYN1UhmJzyS1XX3fZKbkeUcpJe9Sy3FE
-initial_guardian=befa429d57cd18b7f8a4d91a2da9ab4af05d0fbe
 recipient_address=90F8bf6A479f320ead074411a4B0e7944Ea8c9C1
 chain_id_ethereum=2
 
+# load the .env file with the devent init data
+source .env
+# remove brackets and double quotes, transform json list of strings to CSV with no quotes.
+guardians=$(echo $INIT_SIGNERS | sed -r 's/(\[|\]|\")//g')
```

Comment:
Dirty. I love it.

Reply:
replace this with a cleaner method - just passed the CSV string in from the .env file. Didn't love the dirty.